SeaSharpe_CVGS/SeaSharpe_CVGS/Views/Order/PartialSelectedOrder.cshtml

```diff
@@ -1,76 +1,93 @@
 ﻿@model SeaSharpe_CVGS.Models.Order
+@{
+    Layout = null;
+}
+<link rel="stylesheet" href="~/Content/order.css" type="text/css">
 
-<div>
-    <h4>Selected Order</h4>
-    <dl class="dl-horizontal">
-        <dt>
-            @Html.DisplayNameFor(model => model.Member.Id)
-        </dt>
-        <dd>
-            @Html.DisplayFor(model => model.Member.Id)
-        </dd>
-        <dt>
-            @Html.DisplayNameFor(model => model.OrderPlacementDate)
-        </dt>
-        <dd>
-            @Html.DisplayFor(model => model.OrderPlacementDate)
-        </dd>
-    </dl>
-    <table class="table">
-        <tr>
-            <th>
-                Title
-            </th>
-            <th>
-                Platform
-            </th>
-            <th>
-                Price
-            </th>
-        </tr>
-        @if (Model.Id == -1)
-        {
-            <tr>
-                
-                    @if (TempData["messageDan1"] == "")
-	                {
-                        <td>
-                            No Order Selected
-                        </td>
+    
+<div class="row">
+    <div class="col-lg-12">
+        <h3><b>Selected Order</b></h3>
+        <div class="col-md-6">
+            <dl class="dl-horizontal">
+                <dt>
+                    @Html.DisplayNameFor(model => model.Id)
+                </dt>
+                <dd>
+                    @Html.DisplayFor(model => model.Id)
+                </dd>
+                <dt>
+                    @Html.DisplayNameFor(model => model.OrderPlacementDate)
+                </dt>
+                <dd>
+                    @Html.DisplayFor(model => model.OrderPlacementDate)
+                </dd>
+            </dl>
+            </div>
+        <div class="col-md-6">
+            <dl class="dl-horizontal">
+                <dt>
+                    @Html.DisplayNameFor(model => model.Member.Id)
+                </dt>
+                <dd>
+                    @Html.DisplayFor(model => model.Member.Id)
+                </dd>
+            </dl>
+        </div>
+                <table class="table">
+                    <tr>
+                        <th>
+                            @Html.DisplayNameFor(model => model.OrderItems.First().Game.Name)
+                        </th>
+                        <th>
+                           @Html.DisplayNameFor(model => model.OrderItems.First().Game.Platform)
+                        </th>
+                        <th>
+                            @Html.DisplayNameFor(model => model.OrderItems.First().Game.SuggestedRetailPrice)
+                        </th>
+                    </tr>
+                    @if (Model.Id == -1)
+                    {
+                        <tr>
+
+                            @if (TempData["messageDan1"] == "")

```

Comment:
Would be nice if the text represented the purpose. Also is there a reason this is `TempData` and not `ViewData` (aka `ViewBag`)